### ISSUE TITLE
typo

### ISSUE THREAD

**Issue by thegaryking** [ChristopherLucas/MatchingFrontier#51]
a suggestion from 

Byeongdon Oh <donoh@berkeley.edu>
--
"Ps. I might have found a typo in the MatchingFrontier Vignette ([here](https://projects.iq.harvard.edu/files/frontier/files/using_matchingfrontier.pdf)). You may want to change "frontierEst()" to "estimateEffects()" on page 7."

**Comment by ChristopherLucas** [comment]
The documentation on that website hasn't kept up, especially since Noah took over the repo. I can't remember if we linked to that website anywhere, but I could either delete it entirely or leave a pointer to Noah's git repo?

**Comment by ngreifer** [comment]
Yeah, that whole document is outdated if the user is using the version on the [IQSS GitHub](https://iqss.github.io/MatchingFrontier/) (v4.0.0 or greater). That version has plenty of documentation, but the only documentation that is relevant for it is on that website (so some other materials around the web will be outdated). @thegaryking, someone recently mentioned that the link to install the package on your website refers to an old version, so maybe you should update it. I have been working on the package here and there; do you think it would be okay to put it on CRAN at this point?

**Comment by thegaryking** [comment]
let's update all the websites.  otherwise people will just get tired and
switch to a different method.  I take it that this is the correct link for
everything:  https://iqss.github.io/MatchingFrontier/
@Greifer, Noah ***@***.***> can you update
https://projects.iq.harvard.edu/frontier/home too? and LMK what page on my
website should be changed?
Gary
--
*Gary King* - Albert J. Weatherhead III University Professor - Director,
IQSS <http://iq.harvard.edu/> - Harvard University
GaryKing.org <http://garyking.org/> - ***@***.*** - @KingGary
<https://twitter.com/kinggary> - 617-500-7570 - Assistant
***@***.***>: 617-495-9271

**Comment by ChristopherLucas** [comment]
That makes sense to me.

What if we remove the installation and documentation tabs from that website, then on the front page link to @ngreifer's git repo, which is the most up-to-date? That preserves the link (which we included in the AJPS article) but consolidates documentation so we don't have to update docs in multiple locations.

**Comment by ngreifer** [comment]
Yes, I think that is a good idea (also remove the mailing list, since GitHub issues is a better place for discussion). You could also make it so that the link in the AJPS article automatically redirects to https://iqss.github.io/MatchingFrontier/ and remove that page entirely. I don't have access to that website so I can't update it.

**Comment by ChristopherLucas** [comment]
> You could also make it so that the link in the AJPS article automatically redirects to https://iqss.github.io/MatchingFrontier/

Good idea. I'll do that.

**Comment by thegaryking** [comment]
Also, @Greifer, Noah ***@***.***>, I've added you as an
administrator of https://projects.iq.harvard.edu/frontier. Can you make the
changes?

also LMK what should be changed on anything else including my site.
Gary
--
*Gary King* - Albert J. Weatherhead III University Professor - Director,
IQSS <http://iq.harvard.edu/> - Harvard University
GaryKing.org <http://garyking.org/> - ***@***.*** - @KingGary
<https://twitter.com/kinggary> - 617-500-7570 - Assistant
***@***.***>: 617-495-9271

**Comment by ngreifer** [comment]
Okay, I updated the site to include the relevant links and instructions and removed the mailing list. Make sure everything looks okay.

https://projects.iq.harvard.edu/frontier/home

The website is loading very slowly today.

**Comment by ChristopherLucas** [comment]
Looks good to me, thanks Noah.
